processor/internal/transformer/destination_transformer/transform.go

```diff
@@ -0,0 +1,38 @@
+package destination_transformer
+
+import (
+	"context"
+
+	"github.com/rudderlabs/rudder-server/processor/internal/transformer/destination_transformer/embedded/pubsub"
+	"github.com/rudderlabs/rudder-server/processor/types"
+)
+
+type transformerImpl func(ctx context.Context, clientEvents []types.TransformerEvent) types.Response
```

Comment:
[minir] 'Impl' suffix is a bit confusing here, as this is not a concrete implementation.